src/text_file_functions.cpp

```diff
@@ -303,15 +305,19 @@ pos_size_type AOApplication::get_element_dimensions(QString p_identifier,
 QString AOApplication::get_design_element(QString p_identifier, QString p_file,
                                           QString p_char)
 {
-  QString char_ini_path =
-      get_base_path() + "misc/" + get_chat(p_char) + "/" + p_file;
+  QString char_ini_path = get_base_path() + "misc/" + get_chat(p_char) + "/" + p_file;
+  QString char_ini_fallback = get_base_path() + "misc/" + get_char_name(p_char) + "/" + p_file;
```

Comment:
yo hold up that's new behavior, it shouldn't replace the user's chatbox unless the chat is specifically defined in the get_chat() obtained char.ini option chat=